decidim-meetings/app/views/decidim/meetings/meetings/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <%= render partial: "decidim/shared/component_announcement" %>
 
-<%= cell "decidim/meetings/meetings_map", search.results %>
+<% if current_component.settings.maps_enabled? %>
```

Comment:
Just an idea but could we also hide the map when there are no meetings with map pointers associated to them (online meetings only)?

Reply:
I think it's a good idea, if @decidim/product approves it I'll include the change